### ISSUE TITLE
All custom interactive elements need to be accessible

### ISSUE THREAD

**Issue by toolness** [GovLab/styleguide#27]
As [Alice Bartlett's TXJS 2015 talk attests](https://www.youtube.com/watch?v=Q8Mj7_0Lok0), making accessibility-friendly UI widgets "from scratch" that address all edge cases and are compatible with all major browsers and screen-readers is actually quite time-consuming; I learned this the hard way when working on a ["from scratch" modal component on teach.mozilla.org](https://github.com/mozilla/teach.mozilla.org/search?p=3&q=modal&type=Issues&utf8=%E2%9C%93), which seemed simple enough at first but kept coming back to haunt us as we uncovered new accessibility and usability edge cases.

This is a big part of why we decided to reuse bootstrap's JS plugins as much as possible in NoI: specifically, for [modals](https://github.com/GovLab/noi2/pull/102), [alerts](https://github.com/GovLab/noi2/pull/101), and [tabs](https://github.com/GovLab/noi2/pull/109). While the downside here is that we don't necessarily get the slickest next-generation visual effects, the upside is that we don't leave users on screen readers or other unusual browser configurations in the dust.

**Comment by claudioccm** [comment]
Agreed. I'm curious to see if we have any analytics on screen-reader usage, so I can think in terms or prioritization.

Also, I agree with the NOI strategy, and maybe we should use the same strategy here. 
Picking up a few structural elements from bootstrap might be smart.

On Foundation, we can create a custom build, with just the features we want, or even import them as sass partials. http://foundation.zurb.com/sites/download.html/ Do you know if bootstrap does the same?

Let's talk more about this, and check the PROs and CONs for this. This can be big.
